### PR TITLE
Fix DGL version

### PR DESCRIPTION
**Pull request by ktangsali**
# Modulus Pull Request

## Description
Fixes DGL version in the Dockerfile.

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -73,8 +73,8 @@ ARG DGL_BACKEND=pytorch
 ENV DGL_BACKEND=$DGL_BACKEND
 ENV DGLBACKEND=$DGL_BACKEND
 
-RUN pip install --no-cache-dir --no-deps dgl -f https://data.dgl.ai/wheels/cu121/repo.html
-RUN pip install --no-cache-dir --no-deps dglgo -f https://data.dgl.ai/wheels-test/repo.html
+RUN pip install --no-cache-dir --no-deps dgl==2.0.0 -f https://data.dgl.ai/wheels/cu121/repo.html
+RUN pip install --no-cache-dir --no-deps dglgo==2.0.0 -f https://data.dgl.ai/wheels-test/repo.html
```
Comment (NickGeneva) [inline Dockerfile:77]: dglgo is not 2.0.0, I dont know why we even install this. Its max version is 0.2.0

Comment (NickGeneva) [reply Dockerfile:77]: I only know this because I did the same thing

https://pypi.org/project/dglgo/

**Review by NickGeneva — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (ktangsali) [reply Dockerfile]: Yea, good catch! I removed the version specified from `dglgo`. As far as removing `dglgo` altogether is considered, that is fair. As I understand that's just a CLI interface for DGL. Let me check if we can completely remove it.

Comment (ktangsali) [reply Dockerfile]: Removed DGLGO

**Review by Alexey-Kamenev — APPROVED** [review]

**Pull request merged**